.github/workflows/release-check.yml

```diff
@@ -2,7 +2,7 @@ name: Release file check
 
 on:
   pull_request:
-    types: [opened, synchronize, reopened, ready_for_review, review_requested]
+    types: [opened, synchronize, reopened]
```

Comment:
If we remove this then we don’t get the comment when the PR goes from draft to ready, what if debounced events on the bot instead? 🤔